docs/src/tutorials/tutorial_morphing.md

```diff
@@ -0,0 +1,194 @@
+# **Tutorial 9:** Morphing `Javis` Objects   
+<!-- TODO test all examples once more and update if they dont work -->
```

Comment:
Maybe do this first before trying to shift them in a way to have a better morphing experience. Would like to merge this branch first to keep it a bit smaller.